STATUS: INLINE FAIL SETUPTEARDOWN
MODIFIED: de.strullerbaumann.visualee.examiner.cdi.ExaminerEventTest.testFindEvents
MODIFIED FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/examiner/cdi/ExaminerEventTest.java
CLEANER: de.strullerbaumann.visualee.examiner.ExaminerTest.testCleanPrimitives
CLEANER FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/examiner/ExaminerTest.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.SourcecodeFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/SourcecodeFilterTest.java
ORIGINAL CLEANER SIZE: 45
NEW CLEANER SIZE: 1
ELAPSED TIME: 50.04400014877319
==========================
@@ -113,0 +113,3 @@
+ {
+     FilterContainer.getInstance().clear();
+ }
